src/sanescansrv/server.py

```diff
@@ -339,9 +340,13 @@ def preform_scan(
     """Scan using device and return path."""
     if out_type not in {"pnm", "tiff", "png", "jpeg"}:
         raise ValueError("Output type must be pnm, tiff, png, or jpeg")
-    filename = f"scan.{out_type}"
+    filename = f"{uuid.uuid4()!s}_scan.{out_type}"
     assert app.static_folder is not None
-    filepath = Path(app.static_folder) / filename
+    if not path.exists("/tmp/sanesansrv/"):
+        makedirs("/tmp/sanesansrv/")
+    if not path.exists(Path(app.static_folder + "/scans")):
+        symlink("/tmp/sanesansrv/", Path(app.static_folder + "/scans"))
+    filepath = Path("/tmp/sanesansrv/") / filename
```

Comment:
Should be `sanescansrv` if we wanted to match the project name, but thinking about it a bit more I suggest we use `sane_scan_srv` instead for the folder name.

Also, I would suggest creating paths by joining them, ex
```python
temp_dir = Path("/tmp") / "sanesansrv"
if not temp_dir.exists():
    makedirs(temp_dir)
```

Reply:
I will make that change, I am not sure about windows working. Do you want to revert to not using /tmp if its windows?

Reply:
Doing a bit of research, looks like we should use `%Temp%` if on windows (expands to `C:\Users\{Username}\AppData\Local\Temp`).

Reply:
How should we handle this? Is there a tmp folder variable for linux (or unix) as well? so that we can load both using this variable and default to something if they are not present.

Reply:
On Research there are variables for linux that can be set, but they are meant to override `/tmp` if present. So we can use the url if windows and then use the second url if any other OS? Would that be fine. 

PS: I do not have a windows machine to validate my work :smile: Linux user since 2008.